### PR TITLE
[plugin-mobile-app] Use mobile_app spring profile only if it's selected by vividus mobile related profile

### PR DESCRIPTION
**Pull request by uarlouski**

### CODE REVIEW

**Review by AlenaBudevich — APPROVED** [review]

File: vividus-plugin-mobile-app/src/main/resources/properties/defaults/run.properties [hunk 1]
```diff
@@ -1,5 +1,3 @@
-spring.profiles.active=mobile_app
```
Comment (ikalinin1) [inline vividus-plugin-mobile-app/src/main/resources/properties/defaults/run.properties:1]: what is the reason for these changes?

File: vividus-plugin-mobile-app/src/main/resources/properties/defaults/run.properties [hunk 2]
```diff
@@ -1,5 +1,3 @@
...
-
 bdd.all-meta-filters=groovy: !skip && (${bdd.meta-filters})
 
 mobile-app=true
```
Comment (ikalinin1) [inline vividus-plugin-mobile-app/src/main/resources/properties/defaults/run.properties:3]: What about this one?

**Review by ikalinin1 — CHANGES_REQUESTED** [review]

File: vividus-plugin-mobile-app/src/main/resources/properties/defaults/run.properties (hunk 1 above)
Comment (uarlouski) [reply vividus-plugin-mobile-app/src/main/resources/properties/defaults/run.properties:1]: mobile_app profile is used by default when we add plugin-mobile-app as a dependency, so it should not land in defaults folder

**Review by valfirst — APPROVED** [review]

**Review by ikalinin1 — APPROVED** [review]
